pkg/operator/configobservation/etcd/OWNERS

```diff
@@ -0,0 +1,6 @@
+reviewers:
+  - abhinavdahiya
+  - wking
```

Comment:
hmm, why am I here?  I'm not even sure what this directory is about...

Reply:
Filed #674 removing myself.  I guess we can talk about whether I should be a reviewer/approver in that PR.